_includes/layouts/base.njk

```diff
@@ -5,11 +5,26 @@
 
 	<body>
 
-		<header>
-			<em>
-				<a href="{{ '/' | url }}">{{ metadata.title }}</a>
-			</em>
-			{% include "components/nav.njk" %}
+		<header class="header-main">
```

Comment:
This elements needs a role="banner" https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Banner_Role

Reply:
Amended.